app/hotels/src/allHotels/AllHotels.js

```diff
@@ -17,53 +15,69 @@ type Props = {|
   onFilterChange: (filter: SearchParametersType) => void,
 |};
 
-export default class AllHotelsSearch extends React.Component<Props> {
+export default class AllHotels extends React.Component<Props> {
+  /**
+   * Submit form with the initial date interval parameters.
+   */
+  componentDidMount = () => {
+    this.props.onFilterChange({
+      checkin: moment()
+        .add(1, 'week')
+        .startOf('isoWeek')
+        .format('YYYY-MM-DD'),
+      checkout: moment()
+        .add(1, 'week')
+        .endOf('isoWeek')
+        .format('YYYY-MM-DD'),
+    });
+  };
+
   handleSearchChange = (search: SearchParametersType) => {
     this.props.onFilterChange(search);
   };
 
-  isReadyToSearch = (): boolean => {
-    const { search: s } = this.props;
+  renderInnerComponent = rendererProps => {
+    const {
+      onFilterChange,
+      openSingleHotel,
+      search,
+      location,
+      onLocationChange,
+    } = this.props;
```

Comment:
Note: I am not exactly super big fan of destructions like this because even simple things like `const p = this.props` will save so many characters... 😊